src/Playwright/Core/CDPSession.cs

```diff
@@ -50,12 +50,12 @@ public CDPSession(IChannelOwner parent, string guid) : base(parent, guid)
 
     public async Task DetachAsync()
     {
-        await _channel.DetachAsync();
+        await _channel.DetachAsync().ConfigureAwait(false);
```

Comment:
Since the method just forwards, it doesn't need to use `async` / `await`:

```c#
    public Task DetachAsync()
    {
        return _channel.DetachAsync();
    }
```